scripts/ipfs/install

```diff
@@ -18,29 +30,43 @@ sudo chown root:staff /usr/local/bin/ipfs
 rm -rf "$BASE_DIR/tmp"
 
 # Initialize IPFS
-ipfs init
+ipfs init || true
```

Comment:
Does this occasionally fail? Entropy + timeout?

Reply:
I believe it's in case the `~/.ipfs` folder already exists. On my laptop:
```
❯ ipfs init
initializing IPFS node at /home/makeworld/.ipfs
Error: ipfs configuration file already exists!
Reinitializing would overwrite your keys.
```